guides/common/modules/con_managing-organizations.adoc

```diff
@@ -9,30 +9,3 @@ ifndef::satellite[]
 You can create and manage multiple organizations through {ProjectName}, then divide and assign subscriptions to each individual organization.
 endif::[]
 This provides a method of managing the content of several individual organizations under one management system.
-Here are some examples of organization management:
-
-Single Organization::
-A small business with a simple system administration chain.
-In this case, you can create a single organization for the business and assign content to it.
-
-Multiple Organizations::
-A large company that owns several smaller business units.
-For example, a company with separate system administration and software development groups.
-In this case, you can create organizations for the company and each of the business units it owns.
-This keeps the system infrastructure for each separate.
-You can then assign content to each organization based on its needs.
-
-External Organizations::
-A company that manages external systems for other organizations.
-For example, a company offering cloud computing and web hosting resources to customers.
-In this case, you can create an organization for the company's own system infrastructure and then an organization for each external business.
-You can then assign content to each organization where necessary.
-
-ifndef::orcharhino[]
-A default installation of {ProjectName} has a default organization called *Default Organization*.
-endif::[]
```

Comment:
@maximiliankolb Why was this under an ifndef for orcharhino? Does it mean orcharhino doesn't create the Default Organization?

Reply:
Yes, that's correct. during orcharhino installation, users have to enter their org+loc names.

Reply:
Great, thanks for confirming, and thanks for adding the missing ifndefs to cover this :)